tox.ini

```diff
@@ -25,28 +29,24 @@ basepython =
 
 deps =
     coverage
-    codecov
 
     django3.1: Django>=3.1,<3.2
     django3.2: Django>=3.2,<3.3
     django4.0: Django>=4.0,<4.1
+    django4.1: Django>=4.1,<4.2
     
     wagtail2.15: wagtail>=2.15,<2.16
     wagtail2.16: wagtail>=2.16,<2.17
     wagtail3.0: wagtail>=3.0,<4.0
+    wagtail4.0: wagtail>=4.0,<4.1
```

Comment:
suggestion: now that 4.1 is out, may as well add it here. there should be minimal breaking changes (says he hopeful)

Reply:
Testing against Wagtail 4.1 added and passing.